white-list

```diff
@@ -0,0 +1,22 @@
+10.97.164.88
```

Comment:
why is this file required?

Reply:
a list of site we ignore when checking for broken links in mkdocs